beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -48,6 +49,11 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 	// Step 1 - Sync to end of finalized epoch.
 	for s.chain.HeadSlot() < helpers.StartSlot(highestFinalizedEpoch()+1) {
 		root, finalizedEpoch, peers := bestFinalized()
+		if len(peers) == 0 {
+			log.Warn("No peers; waiting for reconnect")
+			time.Sleep(noPeersWaitTime)
```

Comment:
can we just use the same refresh period as `refereshTime` ? its 6 seconds, so would serve the same purporse

Reply:
Yeah the patches crossed so it wasn't there when I picked my name.  Will use that instead.